dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/PropertyService.java

```diff
@@ -194,6 +207,23 @@ public static String filterURL(String url) {
         sb.append("****");
         return sb.toString();
     }
+    
+    public static String filterConnectionProperties(String props) {
+        final String keyStorePattern = "javax\\.net\\.ssl\\.keyStorePassword\\s*=\\s*(.*?)\\s*(;|$)";
+        final String trustStorePattern = "javax\\.net\\.ssl\\.trustStorePassword\\s*=\\s*(.*?)\\s*(;|$)";
```

Comment:
Rather than having separate patterns/matchers for keyStorePassword=..., you could use a single pattern for just Password=...